redash/handlers/admin.py

```diff
@@ -33,7 +35,7 @@ def outdated_queries():
 
     response = {
         'queries': QuerySerializer(outdated_queries, with_stats=True, with_last_modified_by=False).serialize(),
-        'updated_at': manager_status['last_refresh_at'],
+        'updated_at': manager_status.get('last_refresh_at', datetime.now()),
```

Comment:
What does it fix?

Reply:
It throws 500 errors when Redis is restarted and no manager status was found. (Not relevant to this PR, but fixed as it is minor.)